### PR TITLE
exculded adding test to dist folder

### LINKED ISSUE #45: Add unit tests to utilities
## Description:
Jest unit testing framework has been added to the project. This task is to increase Code Coverage to an acceptable level by...

1. Write unit tests for any utility methods
2. Look at abstracting data access layer in order to mock query methods

### PR DESCRIPTION
**Pull request by tobiaswright**
## Type of PR

- [ ] Documentation changes
- [ ] Code changes
- [x ] Test changes
- [ ] CI-CD changes
- [ ] GitHub Template changes

## Purpose of PR
Review test added to existing test suite

## Review notes
We may need to add some logging, but will surface in review

## Issues Closed or Referenced

- Closes #45  (this will automatically close the issue when the PR closes)
- References #<issue number> (this references the issue but does not close with PR)

### CODE REVIEW

File: src/utilities/queryUtilities.ts [hunk 1]
```diff
@@ -11,8 +10,9 @@ export class QueryUtilities {
 
         if ( id.length > 5 && (id.startsWith("tt") || id.startsWith("nm"))) {
             idInt = parseInt(id.substring(2), 10);
+            return (idInt % 10).toString();
+        } else {
```
Comment (aflinchb) [inline src/utilities/queryUtilities.ts]: Although unexpected, we should handle the case of something like "ttttttt".  (This is me remembering why I had the ternary operation in here before, to return an empty string or "0" instead of NaN.)

File: src/utilities/queryUtilities.ts [hunk 2]
```diff
@@ -1,7 +1,6 @@
 /**
  * Utilities for querying from CosmosDB.
  */
-
 export class QueryUtilities {
 
     // Compute the partition key based on the movieId or actorId
```
Comment (aflinchb) [inline src/utilities/queryUtilities.ts]: It will be good to call out in the comments that we are defaulting to 0 for the genre direct read scenario, since all genre documents have partition key = "0"

Comment (aflinchb) [inline src/utilities/queryUtilities.ts]: It will be good to call out in the comments that we are defaulting to 0 for the genre direct read scenario, since all genre documents have partition key = "0"

File: src/utilities/__tests__/utils.tests.ts [hunk 3]
```diff
@@ -1,5 +1,25 @@
...
+import { DateUtilities } from "../dateUtilities";
+import { IsEqualToProperty } from "../validationUtilities";
+import { VersionUtilities } from "../versionUtilities";
 
 test("Get partition key", () => {
+  expect(QueryUtilities.getPartitionKey("nm1234")).toEqual(expect.any(String));
+  expect(QueryUtilities.getPartitionKey("nm1234")).toBe("4");
   expect(QueryUtilities.getPartitionKey("tt1234")).toBe("4");
+  expect(QueryUtilities.getPartitionKey("tt1")).toBe("0");
 });
```
Comment (aflinchb) [inline src/utilities/__tests__/utils.tests.ts]: We should add a test for a genre, like: getPartitionKey("Action").toBe("0") and also test the edge case "ttttttt".

File: src/utilities/queryUtilities.ts (hunk 1 above)
Comment (tobiaswright) [reply src/utilities/queryUtilities.ts]: done. also removed the else statement as it's not needed

File: src/utilities/queryUtilities.ts (hunk 2 above)
Comment (tobiaswright) [reply src/utilities/queryUtilities.ts]: done

File: src/utilities/__tests__/utils.tests.ts (hunk 3 above)
Comment (tobiaswright) [reply src/utilities/__tests__/utils.tests.ts]: Added additional test

**Comment by tobiaswright** [comment]
Update made to DockerCI
